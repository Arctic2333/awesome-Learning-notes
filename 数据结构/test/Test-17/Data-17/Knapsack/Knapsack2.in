 5 10
 2 2 6 4 5
 6 3 5 4 6

