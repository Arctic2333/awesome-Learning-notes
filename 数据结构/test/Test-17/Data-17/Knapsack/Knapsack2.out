Options: 1 1 0 0 1 
Max Value: 15
